src/makeChange.js

```diff
@@ -1,3 +1,63 @@
+let keys = {
```

Comment:
`keys` is not a great name for this object.  How about `coins` or `denominations`?